test/tools/d_do_test.d

```diff
@@ -1606,7 +1606,7 @@ int runBashTest(string input_dir, string test_name, const ref EnvData envData)
         const cmd = "bash " ~ script ~ ' ' ~ input_dir ~ ' ' ~  test_name;
         const env = [
             // Make sure the path is bash-friendly
-            "DMD": envData.dmd.relativePath(dmdTestDir).replace('\\', '/')
+            "DMD": envData.dmd.relativePath(dmdTestDir).replace("\\", "/")
```

Comment:
This makes me suspicious ... how come that this change is required if the build is supposed to use the same version as the auto-tester?

Reply:
auto-tester uses the Makefile to run the testsuite.
Example:

```
HOST_DC=/home/braddr/sandbox/at-client/release-build/dmd-2.079.0/windows/bin/dmd.exe
HOST_DC=D:\sandbox\at-client\release-build\dmd-2.079.0\windows\bin\dmd.exe
cd test
gmake -j4
Makefile:103: Please use HOST_DMD instead of HOST_DC!
Creating output directory: test_results
Building d_do_test tool
OS: 'windows'
MODEL: '32'
PIC: ''
../generated/windows/release/32/dmd.exe -conf= -m32  -g -i -Itools -version=NoMain -lowmem -unittest -run tools/d_do_test.d
1 modules passed unittests
../generated/windows/release/32/dmd.exe -conf= -m32  -g -i -Itools -version=NoMain -lowmem -odtest_results -oftest_results/d_do_test.exe tools/d_do_test.d
../generated/windows/release/32/dmd.exe -conf= -m32  -g -odtest_results -oftest_results/run.exe -i -release run.d
test_results/run.exe --environment
```

and thus uses the currently generated compiler for `d_do_test`.
Azure, however, calls `run.d` directly which uses HOST_DMD for `d_do_test`

Reply:
I see, they were still defaulting to the generated compiler. Nevermind then